src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -1316,6 +1346,26 @@ export default class InviteDialog extends React.PureComponent<IInviteDialogProps
             }
             buttonText = _t("Go");
             goButtonFn = this.startDm;
+            extraSection = <div className="mx_InviteDialog_section_hidden_suggestions_disclaimer">
+                <span>{ _t("Some suggestions may be hidden for privacy.") }</span>
+                <p>{ _t("If you can’t see who you’re looking for, send them your invite link below.") }</p>
```

Comment:
```suggestion
                <p>{ _t("If you can't see who you’re looking for, send them your invite link below.") }</p>
```

Reply:
Serves me right for using the Copy button in Figma